### DISCUSSION TITLE
Partnership with universities

### DISCUSSION THREAD

**Discussion by GeneralVimes** [airsdk/Adobe-Runtime-Support#2184] [General]
Somewhen around 2019 @ajwfrost mentioned the idea to partner with universities to include AIR/Action Script 3 into the curriculum. Do you have any ideas about this now?

**Comment by manishpatel0404** [comment]
How about Authorised Training Centers? I would be glad to offer part of our offices if it helps promote AIR/ActionScipt3. At no cost, of course.

**Reply by GeneralVimes** [reply]
Wow, this is a great idea, too! I just started thinking on a smaller scope first. Like, there is a discipline "Cross-platform development", for which AIR fits ideally, I suppose

**Reply by manishpatel0404** [reply]
IMHO, most Universities would only come on board after a platform is significantly visible in the public space. If driven by passionate training partners, an ATC would be able to improve the public visibility. Of course, Harman would also have to put in significant effort in promoting the ATC.
